### DISCUSSION TITLE
Old Pdf downloaded when we load the new one

### DISCUSSION THREAD

**Discussion by bibinwraj** [stephanrauh/ngx-extended-pdf-viewer#1053] [Q&A]
Initial pdf was loaded, then try to change the pdf src with new pdf content, that time the old pdf getting downloaded with out any button click event. tool bar  download button are not enabled . so how to prevent the pdf from downloading.

**Comment by stephanrauh** [comment]
Sounds like you're using an old version of the PDF viewer. This is a bug I solved  a long time ago. Can you update to version 10?

**Reply by stephanrauh** [reply]
Hm, maybe you've run into this bug: https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/802#issuecomment-981313394

**Comment by bibinwraj** [comment]
I am using  "version": "11.0.0-alpha.7",
